msdk-io/msdk-io-mzml2/src/main/java/io/github/msdk/io/mzml2/MzMLFileMemoryMapper.java

```diff
@@ -0,0 +1,47 @@
+/* 
+ * (C) Copyright 2015-2016 by MSDK Development Team
+ *
+ * This software is dual-licensed under either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+
+package io.github.msdk.io.mzml2;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.RandomAccessFile;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
+
+import javax.annotation.Nonnull;
+
+public class MzMLFileMemoryMapper {
+	private final @Nonnull File mzMLFile;
+
+	public MzMLFileMemoryMapper(String mzMLFilePath) {
```

Comment:
Since you are adding a constructor with String and File, why not also add a constructor with Path (https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html).
That's something we should eventually add to all modules.